### DISCUSSION TITLE
Is the CODEOWNERS feature useful?

### DISCUSSION THREAD

**Discussion by StanczakDominik** [PlasmaPy/PlasmaPy#1348] [General]
I guess this question is mostly aimed at @namurphy @rocco8773 @pheuer. 

Do you find the code owners feature (as setup in https://github.com/PlasmaPy/PlasmaPy/blob/34dbcf6b9a408b9cab5cd641f0c4b8d7374c4586/CODEOWNERS) useful? Should it be adjusted in any way?

I'm asking, because lately I've been un-requesting a whole bunch  of review request for stuff like warning removals (#1344) made for specific people, where anyone could have handled them, really.

Maybe we should just remove lines `5:`, so we just get the plasmapy-reviewers auto-ping on everything?

**Comment by namurphy** [comment]
That's a good question.  That feature is probably the most useful when there are well-defined package maintainers.  Right now we tend to be spread out across subpackages.  I'd be happy to try out having plasmapy-reviewers get pinged with everything, and then we can think about it again after a few months.  Thanks for bringing this up!

**Comment by pheuer** [comment]
In theory I like being auto-requested for any changes to the proton radiography module (actually, I'd like to add Thomson to that too). No one other than me submits anything there, but if they did... 

Maybe the solution is to keep these, but only for specific cases where a specific "code owner" can be identified (like proton radiography), but otherwise just ping plasmapy-reviewers for everything else?
